lib/clif-backend/Cargo.toml

```diff
@@ -9,14 +9,14 @@ edition = "2018"
 
 [dependencies]
 wasmer-runtime-core = { path = "../runtime-core", version = "0.5.7" }
-cranelift-native = { version = "0.31" }
-cranelift-codegen = { version = "0.31" }
-cranelift-entity = { version = "0.31" }
-cranelift-frontend = { package = "wasmer-clif-fork-frontend", version = "0.31" }
-cranelift-wasm = { package = "wasmer-clif-fork-wasm", version = "0.31" }
+cranelift-native = { git = "https://github.com/wasmerio/cranelift.git", rev = "060c5a8e3553540144ca08ca028972c57693fa6e" }
```

Comment:
Can we update the versions of cranelift? (meaning: publishing our forks and use them here).

Reply:
Done.